python/ray/tests/test_runtime_env_complicated.py

```diff
@@ -594,6 +595,43 @@ def f():
             assert ray.get(f.remote())
 
 
+@pytest.mark.skipif(
+    os.environ.get("CI") is None,
+    reason="This test is only run on CI because it uses the built Ray wheel.")
+@pytest.mark.skipif(
+    sys.platform != "linux", reason="This test is only run on Buildkite.")
+def test_env_installation_nonblocking():
+    """Test fix for https://github.com/ray-project/ray/issues/16226."""
+    env = {"env_vars": {"TEST_123": "123"}}
```

Comment:
Is this necessary?

Reply:
(if so leave a comment why?)

Reply:
Good catch, don't think it's necessary to have a runtime_env in the job config--I'll remove it